### PR TITLE
Robotfwk test

### PR DESCRIPTION
**Pull request by jbgury**
Branch to create functional tests

### CODE REVIEW

File: templates/account/login.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <meta name="author" content="">
     <link rel="shortcut icon" href="{{ STATIC_URL }}images/favicon.png">
 
-    <title>{% trans  "Signin on Libreosto" %}</title>
+    <title>{% trans  "Signin on Libreosteo" %}</title>
```
Comment (JocelynDelalande) [inline templates/account/login.html:12]: I think that require a change in translations file

**Review by JocelynDelalande — COMMENTED** [review]
Interesting approach, never seen that kind of tests before :+1:

File: templates/account/login.html (hunk 1 above)
Comment (jbgury) [reply templates/account/login.html:12]: Yes you're right, i try tout execute tests before 😊

**Comment by jbgury** [comment]
Success at last.... on python 2.7 and python 3.6.
I have to complete some tests before merging on develop

File: templates/account/login.html (hunk 1 above)
Comment (jbgury) [reply templates/account/login.html:12]: fixed on the branch

**Pull request merged**